.env.template

```diff
@@ -14,7 +14,7 @@ DB_PASSWORD=$DB_PASSWORD
 BROADCAST_DRIVER=log
 CACHE_DRIVER=file
 QUEUE_DRIVER=sync
-SESSION_DRIVER=cookie
+SESSION_DRIVER=database
 SESSION_LIFETIME=1440
```

Comment:
Which version of laravel are we using? From what i read, this field now means minutes (and not seconds). 
See: https://stackoverflow.com/questions/41983976/laravel-5-session-lifetime

Reply:
yes it is around 24 mins. This is a standard once.

Reply:
@gsluthra if you want to change it. Please suggest how much it should be

Reply:
Laravel Framework 8.83.27

Reply:
25 mins is fine. Just want to be sure that 1440 = 24 mins.. and not 1440 mins? Like I said, newer versions of LARAVEL treat SESSION_LIFETIME in minutes and not seconds. Can you verify that crater is expiring the session after 25 mins of inactivity? (@atish160384 )

Reply:
ok